### DISCUSSION TITLE
How do I remove/clear a "datetime" value?

### DISCUSSION THREAD

**Discussion by jobee** [tinacms/tinacms#4041] [Help]
Field config:

```typescript
{
    type: "datetime",
    name: "date",
    label: "Date",
    ui: {
        timeFormat: "HH:mm"
    },
},
```

Once set, the frontmatter is:

```yaml
---
date: 2023-06-26T10:35:31.328Z
---
```

Tina lets me remove the date and hit the save button, but won't save it (still showing "Unsaved Changes").
<img width="1086" alt="Screenshot 2023-06-26 at 12 56 08" src="https://github.com/tinacms/tinacms/assets/5636715/03804894-7972-4125-9e8e-9ea5e3d7c572">

Expected behaviour: Empty frontmatter
```yaml
---

---
```

**Comment by 3stepwin** [comment]
test

**Comment by Calinator444** [comment]
Hey guys, 

Dates can be cleared using the reset button on the date time picker in the TinaCMS editor. This feature was added for TinaCMS 3.0.0 after we overhauled the date time picker, so if you're on an older Tina version you'll need to upgrade to use this.

Please let me know if you need anything else.

Thanks

<img width="1877" height="199" alt="CleanShot 2026-03-17 at 08 13 35" src="https://github.com/user-attachments/assets/ca611c80-3308-441b-9eb0-a3b19a8b2ff7" />
